code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -624,13 +624,18 @@
 	desc = "A special box for sensitive people."
 	icon_state = "hugbox"
 	foldable = null
+	var/cooldown = 0
 
 /obj/item/weapon/storage/box/hug/attack_self(mob/user)
-	..()
-	user.changeNext_move(CLICK_CD_MELEE)
-	playsound(loc, "rustle", 50, 1, -5)
-	user.visible_message("<span class='notice'>[user] hugs the [src].</span>","<span class='notice'>You hug the [src].</span>")
-	return
+	if (cooldown < world.time)
+		cooldown = (world.time + 10) // Sets cooldown at 1 second
+		..()
+		user.changeNext_move(CLICK_CD_MELEE)
+		playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -5)

```

Comment:
It is, actually.

On 21 July 2015 at 13:41, Iamgoofball <notifications@github.com> wrote:

> In code/game/objects/items/weapons/storage/boxes.dm
> <https://github.com/tgstation/-tg-station/pull/10734#discussion_r35063174>
> :
>
> >
> >  /obj/item/weapon/storage/box/hug/attack_self(mob/user)
> > -	..()
> > -	user.changeNext_move(CLICK_CD_MELEE)
> > -	playsound(loc, "rustle", 50, 1, -5)
> > -	user.visible_message("<span class='notice'>[user] hugs the [src].</span>","<span class='notice'>You hug the [src].</span>")
> > -	return
> > +	if (cooldown < world.time)
> > +		cooldown = (world.time + 10) // Sets cooldown at 1 second
> > +		..()
> > +		user.changeNext_move(CLICK_CD_MELEE)
> > +		playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -5)
>
>  thudswoosh
> not the hug sound
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/10734/files#r35063174>.
>